src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -48,11 +49,27 @@ const GordonSchedulePanel = ({ profile, myProf }: Props) => {
     });
   }, [profile.AD_Username]);
 
+  useEffect(() => {
+    const getAccordionState = localStorage.getItem('accordionState');
+    if (getAccordionState !== null) {
+      setAccordionState(getAccordionState === 'true');
+    }
+  }, []);
+
+  const toggleAccordionState = () => {
+    setAccordionState(!accordionState);
```

Comment:
I believe react states can have some strange synchronization issues when trying to invert variables like this. The recommended way to avoid those is to use a callback inside of the set function like so.
```suggestion
    setAccordionState(a => !a);
```
Correct, @EjPlatzer ?

Reply:
Thank you for the input, I updated the code with your suggestion.